text/0000-safe-cancel.md

```diff
@@ -0,0 +1,82 @@
+
+- Title: safe-cancel
+- Authors: [John Tromp](mailto:john.tromp@gmail.com)
+- Start date: Oct 10, 2019
+- RFC PR: Edit if merged: [mimblewimble/grin-rfcs#0000](https://github.com/mimblewimble/grin-rfcs/pull/0000)
+- Tracking issue: [Edit if merged with link to tracking github issue]
+
+---
+
+## Summary
+[summary]: #summary
+
+Allow for safe cancellation of pending transactions, preventing future so-called play attacks.
+
+## Motivation
+[motivation]: #motivation
+
+A wallet cannot simply cancel a pending transaction bu forgetting about it and returning its inputs to the wallet balance.
```

Comment:
typo -> bu

Reply:
fixed

Reply:
Linked text doesn't show the fixed version